src/physics/circular_filament.rs

```diff
@@ -16,12 +19,9 @@ use crate::{MU0_OVER_4PI, MU_0};
 ///
 /// # Arguments
 ///
-/// * `ifil`: (A) current in each filament, length `m`
-/// * `rfil`:    (m) r-coord of each filament, length `m`
-/// * `zfil`:    (m) z-coord of each filament, length `m`
-/// * `rprime`:  (m) r-coord of each observation point, length `n`
-/// * `zprime`:  (m) z-coord of each observation point, length `n`
-/// * `out_psi`: (Wb) or (H-A) or (T-m^2) or (V-s), poloidal flux at observation locations, length `n`
+/// * `rzifil`:  (m, m, A-turns) current, r-coord, and z-coord of each filament, length `m`
```

Comment:
The terms should be listed in the docs in the same order they are in the tuple.
Ditto for the other docs of `rzifil` in this file.
```suggestion
/// * `rzifil`:  (m, m, A-turns) r-coord, z-coord, and current of each filament, length `m`
```